docs/releasing.md

```diff
@@ -2,21 +2,26 @@
 
 Releasing, i.e. crate publishing, has been automated via GitHub Actions.
 
-In order to author a new release, you simply tag the desired revision and push
-the resulting tag.
+We use the [`cargo release`](https://github.com/sunng87/cargo-release)
+subcommand to ensure correct versioning. Install via:
 
-**Before releasing** ensure `CHANGELOG.md` is updated appropriately as well as
-`Cargo.toml`.
+```
+$ cargo install cargo-release
+```
+
+**Before releasing** ensure `CHANGELOG.md` is updated appropriately.
 
 ## Process
 
-Please ensure you follow the correct format when creating new tags. For
-instance:
+Using `cargo-release` we can author a new minor release like so:
```

Comment:
I'm not thinking of the new releases as "minor", so perhaps it would be useful to remind that this means `0.(N+1)` in our context?

Reply:
Sure that makes sense to me. Maybe we should simply link to the CHANGELOG where that's described?

Reply:
Sure, but CHANGELOG doesn't mention "minor" releases either. Technically, "minor" is the correct term with regards to semver, but in context of sqlparser using it without mentioning semver is slightly weird. Do what you think is best, I don't have a strong opinion on if or how this should be addressed.

Reply:
I'll leave this as-is for now.